coralproject/talk
# Added translation for email's subject

## What does this PR do?
Added translation for the subjects of the email received when trying to reset your password.

## How do I test this PR?
Check the subject of the email received when you try to reset your user's password.
